rts/Game/UnsyncedGameCommands.cpp

```diff
@@ -1143,8 +1143,13 @@ class SpecFullViewActionExecutor : public IUnsyncedActionExecutor {
 public:
 	SpecFullViewActionExecutor() : IUnsyncedActionExecutor(
 		"SpecFullView",
-		"Sets or toggles between full LOS or ally-team LOS if the local user is a spectator"
-	) {
+		"Sets or toggles LOS settings for current spectator. Fullview: See everything, otherwise visibility is determined by the current team. Fullselect: Whether all units can be selected",
```

Comment:
IMO "if the local user is a spectator" was a bit better wording than "for current spectator" because "current" somewhat implies you can access other spectators somehow.